spec/support/authentication_steps.rb

```diff
@@ -0,0 +1,9 @@
+module AuthenticationSteps
+  def and_i_sign_in(user)
```

Comment:
You could use the warden helper `login_as(@user, scope: user)` instead of this. See dashboard_spec.rb:35

It doesn't go through the pages, but since we're not testing those, nor are we trying to do anything tricky like log in or out part way through — I'd suggest we don't need to include it in our feature tests.

Plus, it'll make them faster! :grinning: